parametric.html

```diff
@@ -1,5 +1,6 @@
 <!DOCTYPE html>
 <html>
+  <!
```

Comment:
Just delete all this code instead of commenting it out. Also, a comment is `<!--` ... `-->`